apps/src/templates/teacherDashboard/TeacherDashboard.jsx

```diff
@@ -163,3 +167,6 @@ const styles = {
     paddingTop: 10
   }
 };
+
+export const UnconnectedTeacherDashboard = TeacherDashboard;
```

Comment:
I know you didn't change this but do you know why we export an `Unconnected` version of this when we don't use redux or radium here?

Reply:
It looks like it was added here: https://github.com/code-dot-org/code-dot-org/commit/9e92a9e1f256f37f4fb1d26fcff5e74bafc57f48 and was necessary then, but isn't now. Shall I remove it and see if anything weird breaks?

Reply:
Thanks for cleaning this up!